client/packages/common/src/ui/components/inputs/TextInput/numeric/PositiveNumberInput.tsx

```diff
@@ -17,7 +17,7 @@ export const PositiveNumberInput = React.forwardRef(
       disabled = false,
       value,
       min = 1,
-      max = Number.MAX_SAFE_INTEGER,
+      max = NumUtils.MAX_SAFE_API_INTEGER,
```

Comment:
So what's wrong with just using `Infinity`?

Reply:
there is no special infinity number in an unsigned int :-)